### PR TITLE
Updated testing/* articles to Symfony 4

### PR DESCRIPTION
**Pull request by javiereguiluz**

### CODE REVIEW

File: testing.rst [hunk 1]
```diff
@@ -158,22 +158,26 @@ As an example, a test could look like this::
...
 
-    To run your functional tests, the ``WebTestCase`` class bootstraps the
-    kernel of your application. In most cases, this happens automatically.
-    However, if your kernel is in a non-standard directory, you'll need
-    to modify your ``phpunit.xml.dist`` file to set the ``KERNEL_DIR``
-    environment variable to the directory of your kernel:
+    To run your functional tests, the ``WebTestCase`` class needs to know which
+    is the application kernel to bootstrap it. The kernel class is usually
+    defined in the ``KERNEL_CLASS`` environment variable (included in the
+    default ``phpunit .xml-dist`` file provided by Symfony):
```
Comment (yceruto) [inline testing.rst]: `phpunit.xml.dist`

File: testing.rst [hunk 2]
```diff
@@ -158,22 +158,26 @@ As an example, a test could look like this::
...
 
     .. code-block:: xml
 
         <?xml version="1.0" charset="utf-8" ?>
         <phpunit>
             <php>
-                <server name="KERNEL_DIR" value="/path/to/your/app/" />
+                <!-- the value is the FQCN of the application kernel -->
+                <server name="KERNEL_CLASS" value="App\Kernel" />
```
Comment (yceruto) [inline testing.rst]: now should be `<env ... />`? https://github.com/symfony/recipes/blob/master/phpunit/phpunit/4.7/phpunit.xml.dist#L12

File: testing/database.rst [hunk 3]
```diff
@@ -113,44 +113,32 @@ not to overwrite data you entered when developing the application and also
...
-        # app/config/config_test.yml
+        # config/packages/test/doctrine.yaml
         doctrine:
             # ...
             dbal:
-                host:     localhost
-                dbname:   testdb
-                user:     testdb
-                password: testdb
+                url: 'mysql://USERNAME:PASSWORD@127.0.0.1/DB_NAME?charset=utf8mb4&serverVersion=5.7'
```
Comment (yceruto) [inline testing/database.rst]: As Doctrine configuration has defined an environment variable (`'%env(resolve:DATABASE_URL)%'`) for all env, I guess we should do the changes in `phpunit.xml.dist` file, adding `<env name="DATABASE_URL" value="..."/>`? then this whole configuration will be gone.

Comment (javiereguiluz) [reply testing/database.rst]: Much better. Thanks!

**Pull request merged**